lesson_13/homework_13/task_2_json.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+from logger import setup_logger
+from pathlib import Path
+
+# Ініціалізація логера
+logger = setup_logger()
+
+def validate_json(file_path):
```

Comment:
```suggestion
def is_validate_json(file_path):
```
для повернення bool прийнято називати з `is_`